README.md

```diff
@@ -1,2 +1,26 @@
 # auth-code-notification
+
+### Overview
 Service used to send email notifications when new auth code requested.
+
+### Requirements
+In order to run the service locally you will need the following:
+- [Java 11](https://www.oracle.com/java/technologies/downloads/#java11)
+- [Maven](https://maven.apache.org/download.cgi)
+- [Git](https://git-scm.com/downloads)
+
+### Getting started
+To check out and build the service:
+1. Clone [Docker CHS Development](https://github.com/companieshouse/docker-chs-development) and follow the steps in the README.
+2. Run ./bin/chs-dev modules enable overseas-entities
```

Comment:
Should this be auth-code-notification? If not could we add a note about dependency.

Reply:
the docker file has been added under the overseas-entities module, i was wondering if we needed overseas-entities running as well for test data, but thinking about it we could have auth-code-notification in its own module. Another change would be needed to separate into its own module in the docker repo.

Reply:
It is part somehow of the update journey, but it could be separated if used with other services